src/js/utm_params.js

```diff
@@ -0,0 +1,9 @@
+const WORKWITHUS_URL = "https://lp.growthtools.com/apply-course?utm_source=";
+const WORKWITHUS_UTM_SOURCE = window.location.pathname == "/" ? "GT-Website-Nav" : window.location.pathname;
+const WORKWITHUS_UTM_CONTENT = "?utm_content=";
+const CONCAT = WORKWITHUS_URL + WORKWITHUS_UTM_SOURCE + WORKWITHUS_UTM_CONTENT;
+
+const WORKWITHUS_HREF_MOBILE = CONCAT + "mobile-menu-link";
+const WORKWITHUS_HREF_DESKTOP = CONCAT + "work-with-me-button";
+
```

Comment:
I would suggest exporting a function or two instead of the already computed href values. The functions could take a `utm_source` param and return the value of `WORKWITHUS_HREF_{MOBILE,DESKTOP}`.

Reply:
@codegoalie I cleaned this up a lot. I left the generateUTMsource function. I'd love more feedback on that if you have any. I may have added too much functionality that is unnecessary, but I added comments with my intention.